### ISSUE TITLE
About the result on validation set

### ISSUE THREAD

**Issue by anno192** [abhinanda-punnakkal/BABEL#7]
Hi, would you mind providing the result of your reported model on the validation set for reference?

**Comment by abhinanda-punnakkal** [comment]
The results are given [here]( https://github.com/abhinanda-punnakkal/BABEL/tree/main/action_recognition#pre-trained-models).
